scripts/ci/requirements.txt

```diff
@@ -29,6 +29,7 @@ datadog
 dill
 distributed
 docker-py
+elasticsearch>=5.0.0
```

Comment:
This hook only supports ElasticSearch version >= 5.0? ES version info should be included in the ElasticSearch Hook documentation. Also it's better to use `elasticsearch>=5.0.0,<6.0.0`.